src/styles/blocks/_calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  position: absolute;
```

Comment:
you don't need to use position absolute, try to use flex on the body to center your calendar